modules/serverless-sinkbinding-kn.adoc

```diff
@@ -0,0 +1,134 @@
+// Module included in the following assemblies:
+//
+// serverless/knstive_eventing/serverless-sinkbinding.adoc
+
+[id="serverless-sinkbinding-kn_{context}"]
+= Using SinkBinding with the Knative CLI (`kn`)
+
+This guide describes the steps required to create, manage and delete a SinkBinding instance using `kn` commands.
+
+.Prerequisites
+
+* You have Knative Serving and Eventing installed.
+* You have the `kn` CLI installed.
+
+.Procedure
+
+. To check that SinkBinding is set up correctly, create a Knative event display service, or event sink, that dumps incoming messages to its log.
++
+You can create the service by entering the following command:
```

Comment:
Do you need these substeps? 
...create a Knative event display service, or event sink, that dumps incoming messages to its log by entering the following command: